git-proxy/values.yaml

```diff
@@ -21,6 +21,7 @@ blob:
   #    environment:
   #    useInstanceMetadata:
   #    useFederatedToken:
+  azureCredsSecretName:
```

Comment:
default value should be the secret name we are already creating in this chart? (and use the same key for the secret being created)

Reply:
I can rename, but azureCredsSecretName == nil means don't use azure creds secret created in wiz-outpost-config. 
I made it look and function like outpost-csi-files.

Reply:
I will add a createAzureCredsSercret flag

Reply:
thought about it and adding createAzureCredsSercret or setting the secret name breaks backwards compatibility